src/css/sumo.scss

```diff
@@ -296,6 +296,27 @@ html[data-theme='light'] {
 }
 
 // SIDEBAR Links
+.menu__list-item {
+  .menu__link {
+    padding: 8px 16px;
```

Comment:
reduce spacing

```suggestion
    padding: 5px 10px;
```

Reply:
I can definitely update this. That said, the 8px grid is a pretty formidable concept in UI/UX design and development.
https://spec.fm/specifics/8-pt-grid (8pt grid section)

That said, I'll update.